client/src/services/nestSeasonUnderPlayer.js

```diff
@@ -0,0 +1,31 @@
+const nestSeasonUnderPlayer = (season) => {
```

Comment:
i would love for this logic to be managed on the backend, prior to sending to the frontend, but it is understandable if things are too tangled to do such a big refactor